src/script/page/preferences/AccountPreferences.tsx

```diff
@@ -185,6 +186,12 @@ const AccountPreferences: React.FC<AccountPreferencesProps> = ({
                 readOnly
               />
             ))}
+            <AccountLink
+              label={t('preferencesAccountLink')}
+              value={`${Config.getConfig().URL.ACCOUNT_BASE}/user-profile/?id=${selfUser.id}`}
+              readOnly
+              data-uie-name="profile-link"
```

Comment:
```suggestion
              data-uie-name="do-copy-profile-link"
```
@see https://github.com/wireapp/wire-webapp/wiki/data-attributes

Reply:
updated